requirements/base.in

```diff
@@ -16,5 +16,5 @@ rapidfuzz
 backoff
 typing-extensions
 # NOTE(jennings): pinned due to later versions not supporting api_key_auth in UnstructuredClient
-unstructured-client>=0.15.1
```

Comment:
@rbiseck3, do you happen to remember why this version was restricted? The newer parameters of the client aren't accessible with this restricted